### PR TITLE
Feature/edit profile styling

### PR DESCRIPTION
**Pull request by collinkallery**
### Type of change made:

- [X] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Styling (Styling to new or existing files)

### Detailed Description
Styling for editProfile component 
Methods tracking inputs, change the state 
Broken method to post / mutate data - attempted to figure it out but we're heading into code-freeze at this point. 

### Why is this change required? What problem does it solve?
Allows a user to see editable versions of their profile information. Would in theory allow a user to modify the data and save it for future user. 

### Was any previously written code changed?
No

### CODE REVIEW

File: code/web/src/modules/pages/EditProfile.js [hunk 1]
```diff
@@ -44,13 +38,76 @@ class EditProfile extends Component {
...
+    e.preventDefault();
+    console.log("hello");
+    this.props.updateUserInformation(this.state)
+  }
+
+  handleInput = (e) => {
+    const stateKey = e.target.id;
+    this.setState({[stateKey]: e.target.value})
+  }
+
```
Comment (collinkallery) [inline code/web/src/modules/pages/EditProfile.js:51]: Method for saving profile information, and method for tracking the input of a user and setting it to state

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -137,7 +137,7 @@ export function updateUserInformation(userDetails) {
   }
 }
 
-const editProfileSetLocalStorageAndCookie = (user) => {
+export function editProfileSetLocalStorageAndCookie(user) {
   const userObject = {email: user.email, name: user.name, role: 'user', shippingAddress: user.shippingAddress}
```
Comment (Maxwell-Baird) [inline code/web/src/modules/user/api/actions.js:141]: Does it matter if the role is set to user and not USER like in the prebuilt seeds?

**Review by Maxwell-Baird — APPROVED** [review]

File: code/web/src/modules/user/api/actions.js (hunk 2 above)
Comment (collinkallery) [reply code/web/src/modules/user/api/actions.js:141]: @Maxwell-Baird That shouldn't matter, the code that was set up prior in the codebase didn't seem to have an issue with that. I can look really quick to see if it makes a difference.

File: code/web/src/modules/pages/EditProfile.js [hunk 3]
```diff
@@ -44,13 +38,76 @@ class EditProfile extends Component {
     }
   }
 
+  saveProfile = (e) => {
+    e.preventDefault();
+    console.log("hello");
+    this.props.updateUserInformation(this.state)
+  }
+
+  handleInput = (e) => {
```
Comment (colintalex) [inline code/web/src/modules/pages/EditProfile.js:47]: I don't know what's happening but this looks impressive! haha

File: code/web/src/modules/pages/EditProfile.js [hunk 4]
```diff
@@ -44,13 +38,76 @@ class EditProfile extends Component {
...
+            </Input>
+          </div>
+          <div style={{display: 'flex', flexDirection: 'row', margin: '2%', justifyContent: 'center'}}>
+            <label style={{display: 'flex', alignItems: 'center'}} htmlFor="email">Email: </label>
+            <Input
+              style={{color: 'white', marginLeft: '2%'}}
+              onChange={this.handleInput}
+              id="email"
+              type="text"
+              value={this.state.email}>
```
Comment (colintalex) [inline code/web/src/modules/pages/EditProfile.js:86]: I presume this is the form that autopops the pre-exisiting info?

**Review by colintalex — APPROVED** [review]

**Pull request merged**